kolibri/plugins/html5_app_renderer/assets/src/views/index.vue

```diff
@@ -83,6 +83,8 @@
     height: 100vh
     max-height: calc(100vh - 24em)
     min-height: 400px
+    overflow-x: auto
+    -webkit-overflow-scrolling: touch
```

Comment:
Yeah, was just copying from http://stackoverflow.com/questions/23083462/how-to-get-an-iframe-to-be-responsive-in-ios-safari#answer-23123845